nncf/torch/pruning/filter_pruning/algo.py

```diff
@@ -620,7 +620,7 @@ def _apply_binary_mask_to_module_weight_and_bias(module: torch.nn.Module,
             types_to_apply_mask.append('batch_norm')
 
         pruned_node_modules = list()
-        for node in graph.get_all_nodes():
+        for node in graph.get_all_eval_nodes():
```

Comment:
keep using eval nodes in all algorithms. compression for training-only nodes is required for NAS (width propagation)